crashtracker/src/configuration.rs

```diff
@@ -6,19 +6,17 @@ use std::time::Duration;
 
 #[repr(C)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
-pub enum CrashtrackerResolveFrames {
-    Never,
-    /// Resolving frames in process is experimental, and can fail/crash
-    ExperimentalInProcess,
-    InReceiver,
+pub enum CrashtrackerStacktraceCollectionOptions {
+    DontCollectStacktrace,
+    CollectStacktraceButDoNotResolveSymbols,
+    CollectStacktraceAndResolveSymbolsInReceiver,
 }
```

Comment:
I think these options / new API looks good!

Small suggestion in terms of naming (feel free to ignore, naming is hard :P)

```suggestion
pub enum StacktraceCollection {
    Disabled,
    WithoutSymbols,
    Enabled,
}
```

Also, for API users, it may be relevant to document when to use Disabled vs WithoutSymbols vs Enabled. E.g. what are the reasons to not always go with Enabled and forget about the others?

Reply:
Good idea.  Done.